File: Main2_13.java
import java.util.Scanner;

public class Main2_13 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int result = 0;

        for (int i = 0; i < n; i++) {
            int x = sc.nextInt();
            boolean prime = true;

            if (x % 2 == 0) {
                if (x == 2) {
                    prime = true;
                } else {
                    prime = false;
                }
            } else {
                for (int j = 3; j * j <= x; j = j + 2) {
                    int r = x % j;
                    if (r == 0) {
                        prime = false;
                        break;
                    }
                }
            }

            if (prime) {
                result++;
            }
        }

        System.out.println(result);
        sc.close();
    }
}

File: Main2_13.py
def main():
    n = int(input())
    result = 0

    for _ in range(n):
        x = int(input())
        prime = True

        if x % 2 == 0:
            if x == 2:
                prime = True
            else:
                prime = False
        else:
            j = 3
            while j * j <= x:
                if x % j == 0:
                    prime = False
                    break
                j += 2

        if prime:
            result += 1

    print(result)


if __name__ == "__main__":
    main()